Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:14.04
 RUN apt-get update &&\
-    apt-get install -y python python-virtualenv git python-dev python-pip python-setupdocs sqlite3 nginx libsasl2-dev libldap2-dev libssl-dev redis-server &&\
+    apt-get install -y python python-virtualenv git python-dev python-pip python-setupdocs sqlite3 nginx libsasl2-dev libldap2-dev libssl-dev redis-server \
+    binutils libproj-dev gdal-bin &&\
```

Comment:
It' doesn't, but Tastypie does. This is temporary fix for ```django.contrib.gis.gdal.error.GDALException```, till [Changed contrib.gis to raise ImproperlyConfigured if gdal isn't installed.](https://code.djangoproject.com/ticket/28178)